src/DGtal/shapes/SurfaceMesh.h

```diff
@@ -161,12 +161,13 @@ namespace DGtal
     /// @tparam RealPointIterator any forward iterator on RealPoint.
     /// @tparam VerticesIterator any forward iterator on the range of vertices defining a face.
     ///
-    /// @param itPos,itPosEnd a range of iterators pointing on the positions of all the
-    /// vertices of the mesh.
+    /// @param itPos start of range of iterators pointing on the positions of vertices of the mesh
```

Comment:
Weird. When you look at https://www.doxygen.nl/manual/commands.html#cmdparam
it says that you can comment multiple parameters using comma separated items.

Reply:
Otherwise your modif is fine.

Reply:
Indeed, that's confusing, maybe CI is using an old doxygen version, or a globlal doxygen option is missing? No idea.